geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/key/AbstractScanIntegrationTest.java

```diff
@@ -49,7 +49,7 @@ public void setUp() {
 
   @After
   public void tearDown() {
-    jedis.flushAll();
+    flushAll();
     jedis.close();
   }
```

Comment:
Did we decide not to switch this test to use a Jedis Cluster?

Reply:
it looks to me like it's using JedisCluster, but maybe I'm missing something